Refactor/#2 CI

AntonioAldisio opened the pull request:
# Refatoração do CI

## Issue #2 

## Descrição
Realizou-se a refatoração do sistema de integração contínua. A refatoração criou mais etapas no processo, facilitando a depuração em caso de erros, e também fixa uma versão do Ubuntu, diminuindo problemas que poderiam surgir após atualizações do GitHub.



## Tipos de Mudanças
- [X] Refactor (alteração de etapas)
- [ ] Bug fix (alteração ininterrupta que corrige um problema)
- [ ] Nova feature (mudança ininterrupta que adiciona funcionalidade)
- [ ] Breaking change (correção ou recurso que faria com que a funcionalidade existente não funcionasse como esperado)
- [ ] Documentação (adição ou remoção de algum artefato)

## Checklist:
<!--- Passe por todos os pontos a seguir e coloque um x em todas as caixas que se aplicam. -->
<!--- Se você não tiver certeza sobre alguma dessas opções, não hesite em perguntar. Nós estamos aqui para ajudar! -->
- [ ] Meu código segue os padrões de estilo deste projeto.
- [ ] Minha alteração requer uma alteração na documentação.
- [ ] Atualizei a documentação de acordo.

AntonioAldisio committed:
remocao  sonar.testExecutionReportPaths

AntonioAldisio committed:
refatoracao CAPJu Service-User CI

AntonioAldisio committed:
inclusao de lint e alteracao de comando para prettier

AntonioAldisio committed:
remocao de dir

DaviAntonio commented on .github/workflows/CI.yaml:
```diff
@@ -1,36 +1,50 @@
 jobs:
-  build:
+  staticCheck:
     runs-on: ubuntu-latest
```
Acho melhor trocar para `ubuntu-22.04`. O ponteiro `ubuntu-latest` atualmente aponta para o Ubuntu 20.04, mas isso pode mudar em breve.

DaviAntonio committed:
Fixa versão do Ubuntu usada no pipeline de CI

A integração contínua usava a denominação `ubuntu-latest`, que
atualmente aponta para a imagem Ubuntu 20.04 hospedada pelo Github. A
especificação foi modificada para referir-se a uma versão fixa, o Ubuntu
22.04, pois em breve o atalho do Ubuntu 20.04 pode deixar de apontar
para a versão 20.04.

Co-Authored-By: Davi Antônio da Silva Santos <antoniossdavi@gmail.com>
Co-Authored-By: Antonio Aldisio <aldisiofilho@gmail.com>

DaviAntonio committed:
Ignorar arquivos swap do vim

ilus1 reviewed (approved):
:+1:
